django/contrib/admin/filters.py

```diff
@@ -376,37 +487,37 @@ def __init__(self, field, request, params, model, model_admin, field_path):
             (
                 _("Today"),
                 {
-                    self.lookup_kwarg_since: str(today),
-                    self.lookup_kwarg_until: str(tomorrow),
+                    self.lookup_kwarg_since: today,
+                    self.lookup_kwarg_until: tomorrow,
```

Comment:
Moving `str()` calls to the `DateFieldListFilter.get_choices()` sound like a cleanup that can be done in advance :thinking: What do you think?

Reply:
[fbed3f1](https://github.com/django/django/pull/16495/commits/fbed3f1283ba45259a6fe252defc8e9cc5b55d66) pulled it out into it's own commit :+1: